### ISSUE TITLE
Target SDK 25

### ISSUE THREAD

**Issue by m3thm4th** [jroddev/android-oss-release-tracker#10]
There are a lot of Android 7, 8 and 9 devices around. It seems this application does not really need to target such a high SDK.

**Comment by jroddev** [comment]
Target SDK or Minimum SDK?
These are currently set as
```
minSdk 29
targetSdk 33
```

There may be some issues with the storage scopes code recently added for import/export.
I'll need to do some checks.
If everything works fine then I will lower the requirement in the next release

**Comment by m3thm4th** [comment]
Yes I meant minSDK of course, to make the app work on more devices

**Comment by jroddev** [comment]
@m3thm4th 
[0.1.5](https://github.com/jroddev/android-oss-release-tracker/releases/tag/0.1.5) now supports minSDK 27.

I'm having some issues with SDK 25 that I believe to be related to the Emulator and Android Studio, not to the actual application. If someone can test and confirm with 25 I would be happy to relax the restriction. In the meantime I'll keep trying to fix my setup.

**Comment by ShareASmile** [comment]
> If someone can test and confirm with 25

@jroddev , I have android 6 device on api 26 , i can help in testing if that can be helpful.

Currently MinSDK 26 can be tested, later ANDROID 5 can be tested if someone with actual device agrees to test

**Comment by ShareASmile** [comment]
For references how that was done in other Apps
https://github.com/JunkFood02/Seal/issues/27

**Comment by ShareASmile** [comment]
@m3thm4th why not Android 5 or 6 in the issue title? that's possible if tested properly & owner of repo agrees to that

**Comment by jroddev** [comment]
Hey everyone. For `0.1.7` preview I have dropped minSdk support from 27 to 21 (Android 5).

You can test out the support with the preview apk 
[android-oss-release-tracker-release-v0.1.7-preview.apk](https://github.com/jroddev/android-oss-release-tracker/releases/download/0.1.6/android-oss-release-tracker-release-v0.1.7-preview.apk)

Artifact is added along side the [0.1.6](https://github.com/jroddev/android-oss-release-tracker/releases/tag/0.1.6) release.

If you can confirm that things work on any of these lower version I'm happy to drop it. Just want to avoid the situation I drop it too low and people end up downloading a broken app.

**Comment by m3thm4th** [comment]
Seems to be working
